times_reader/transforms.py

```diff
@@ -351,6 +351,55 @@ def has_formulas(table):
     return [table for table in tables if not has_formulas(table)]
 
 
+def remove_empty_tables(
+    config: datatypes.Config,
+    tables: List[datatypes.EmbeddedXlTable],
+) -> List[datatypes.EmbeddedXlTable]:
+    """
+    Return a modified copy of 'tables' where empty tables have been deleted from the list.
+
+    :param tables:      List of tables in EmbeddedXlTable format.
+    :return:            List of non-empty tables in EmbeddedXlTable format.
+    """
+
+    check_list = [
+        datatypes.Tag.comagg,
+        datatypes.Tag.comemi,
```

Comment:
Just curious: why not all tags? Also perhaps this is the kind of info that could also go into the meta/config file for table tags?

Reply:
Yes, I think we should handle it through meta data, i.e. defining what constitutes a non-empty table. Some of the tables define single values and don't have column names, so they should be treated differently.